packages/web/examples/TagCloud/src/index.js

```diff
@@ -61,16 +64,17 @@ const Main = () => (
 													{data.group ? data.group.group_city : ''}
 												</div>
 												<div className="flex wrap meetup-topics">
-													{data.group.group_topics
-														.slice(0, 4)
-														.map(tag => (
-															<div
-																className="meetup-topic"
-																key={tag.topic_name}
-															>
-																{tag.topic_name}
-															</div>
-														))}
+													{data.group
+														&& data.group.group_topics
```

Comment:
`data.group && data.group.group_topics &&  ...`

Reply:
It's `data.group && data.group.group_topics.slice`

Reply:
We can add check for `groupd_topics` as well. Just to be on safe side

Reply:
done